gateway/ln-gateway/Cargo.toml

```diff
@@ -36,7 +36,7 @@ aquamarine = "0.3.0"
 bitcoin = { version = "0.29.2", features = ["serde"] }
 bitcoin_hashes = "0.11.0"
 clap = { version = "4.1.6", features = ["derive", "std", "help", "usage", "error-context", "suggestions", "env"], default-features = false }
-cln-plugin = "0.1.5"
+cln-plugin = "=0.1.7"
```

Comment:
Would put the commit message description here instead.

Reply:
done

Reply:
BTW. How are we ever going to update it now? :D

Reply:
just manually pin it to new version